server/models/shop.js

```diff
@@ -9,4 +8,19 @@ exports.getShops = async () => {
 exports.getShop = async (id) => {
     let shop = await knex.select().from('shops').where({id:id});
     return JSON.parse(JSON.stringify(shop[0]));
+}
+
+exports.getReview = async (shopId) => {
+    let reviews = await knex.select('rating','review').from('reviews').where({shop_id:shopId});
+    return JSON.parse(JSON.stringify(reviews));
+}
+
+exports.writeReview = async (rating,userId,review,shopId) => {
```

Comment:
How different from api.js ?

Reply:
api.js only handles the posted request body then uses the function from line 18 of shop.js to write to database